react_juce/core/TextView.h

```diff
@@ -38,21 +38,27 @@ namespace blueprint
         TextView() = default;
 
         //==============================================================================
-        /** Assembles a Font from the current node properties. */
-        juce::Font getFont()
+        /** Assembles a Font from properties. */
+        static juce::Font getFont(const juce::NamedValueSet &_props)
```

Comment:
Could we remove the underscore here?

Reply:
I think we can, but is it really OK?
If we change `_props` to `props`, it will shadow the member inherited from type `View` and we will get some Clang-Tidy warnings.

Reply:
Good spot. `fontProps` maybe?

Reply:
Good idea! Simple and easy to understand.
I fixed it in c724958.